app/service-cli-bytearray.cpp

```diff
@@ -27,8 +27,14 @@ int main(int argc, char *argv[]) {
   std::string input = std_input.str();
   using marian::bergamot::Response;
 
+  marian::bergamot::ResponseOptions responseOptions;
+  responseOptions.qualityScores = true;
+  responseOptions.alignment = true;
+  responseOptions.alignmentThreshold = 0.2f;
+
   // Wait on future until Response is complete
-  std::future<Response> responseFuture = service.translate(std::move(input));
+  std::future<Response> responseFuture =
```

Comment:
I assume the old behaviour is preserved. If no `ResponseOptions` is provided you'd have a default behavior that emulates the old one?

Reply:
No, defaults are set to off. General consensus among Mozilla and Kenneth is these things are off by default.

Reply:
Old behaviour is preserved though. I will collapse service-cli and bytearray (moving output outside) in a PR behind this.

Reply:
@jerinphilip are we ready to merge now?

Reply:
Squashed and merged. Note that I have gotten rid of the move constructor troubles you are likely to have in windows, I don't have means to test them yet. But most of your patch described in https://github.com/browsermt/bergamot-translator/issues/105#issuecomment-826846522 should be in.